src/dbg/analysis/recursiveanalysis.cpp

```diff
@@ -116,23 +116,23 @@ void RecursiveAnalysis::analyzeFunction(duint entryPoint)
             if(xref.addr)
                 mXrefs.push_back(xref);
 
-            if(!mCp.IsNop() && (mCp.InGroup(CS_GRP_JUMP) || mCp.IsLoop())) //non-nop jump
+            if(!mCp.IsNop() && (mCp.IsJump() || mCp.IsLoop())) //non-nop jump
             {
                 //set the branch destinations
                 node.brtrue = mCp.BranchDestination();
-                if(mCp.GetId() != X86_INS_JMP && mCp.GetId() != X86_INS_LJMP) //unconditional jumps dont have a brfalse
+                if(mCp.GetId() != ZYDIS_MNEMONIC_JMP) //unconditional jumps dont have a brfalse

```

Comment:
Should this also exclude ZYDIS_MNEMONIC_JMP_FAR? The previous logic excluded both unconditional jumps

Reply:
Zydis has the same mnemonic constant for both far and near jumps.

Reply:
In case you're wondering why this is the case: we stick to Intel SDM as close as possible and Intel doesn't treat them as separate instructions either.